src/components/Tabs/TabItem/TabItem.js

```diff
@@ -12,7 +12,14 @@ const { block } = bem({
 const TabItem = props => {
     const { href, isActive, label, ...rest } = props;
     return (
-        <a href={href || '#'} {...rest} {...block(props)}>
+        <a
+            href={href || '#'}
+            {...rest}
+            {...block(props)}
+            role="tab"
+            aria-selected={isActive}
+            tabIndex={0}
```

Comment:
maybe better to have tabIndex taken from the props, and only if it isn't define make it 0? Or simpler yet, move it before `{...rest}` then it can be overwritten

Reply:
Valid point, I think moving it before `rest` should be good